### PR TITLE
travis job consolidation for Core, GDT, and FIAM

### PR DESCRIPTION
**Pull request by paulb777**
Incremental step towards reducing backlogs by merging commonly associated builds

#no-changelog

### CODE REVIEW

**Comment by mikehaney24** [comment]
LGTM for GDT

File: scripts/install_prereqs.sh [hunk 1]
```diff
@@ -90,10 +90,6 @@ case "$PROJECT-$PLATFORM-$METHOD" in
   InAppMessaging-iOS-xcodebuild)
     gem install xcpretty
     bundle exec pod install --project-directory=InAppMessaging/Example --repo-update
-    ;;
-
-  InAppMessagingDisplay-*-xcodebuild)
-    gem install xcpretty
     bundle exec pod install --project-directory=InAppMessagingDisplay/Example --repo-update
```
Comment (wilhuff) [inline scripts/install_prereqs.sh]: This can now be `--no-repo-update`

File: scripts/build.sh [hunk 2]
```diff
@@ -249,9 +249,7 @@ case "$product-$method-$platform" in
         "${xcb_flags[@]}" \
         build \
         test
-  ;;
```
Comment (wilhuff) [inline scripts/build.sh]: InAppMessagingDisplay used to build for iPad. `InAppMessaging-xcodebuild-iOS` above probably needs to become `InAppMessaging-xcodebuild-*`?

File: scripts/install_prereqs.sh [hunk 3]
```diff
@@ -90,10 +90,6 @@ case "$PROJECT-$PLATFORM-$METHOD" in
   InAppMessaging-iOS-xcodebuild)
```
Comment (wilhuff) [inline scripts/install_prereqs.sh]: This needs to be `InAppMessaging-*-xcodebuild`

**Pull request merged**